Record_Play/Player.py

```diff
@@ -0,0 +1,22 @@
+import pyaudio
+import wave
+
+
+class Player:
+    def __init__(self, stream, data, p, chunk, wf):
+        self.stream = stream
+        self.data = data
+        self.p = p
```

Comment:
magic val - samo p nic nie mówi, nadaj jakąś w miarę sensowną nazwe, żeby potem było wiadomo co to jest

Reply:
> 
> 
> magic val - samo p nic nie mówi, nadaj jakąś w miarę sensowną nazwe, żeby potem było wiadomo co to jest